src/modules/common/components/Breadcrumbs.styles.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components'
+
+const opposite = direction => direction === 'ltr' ? 'rtl' : 'ltr'
+export const Wrapper = styled.div`
+  margin: 10px 0;
+  text-align: end;
+  white-space: nowrap;
+  overflow-x: auto;
+  direction: ${props => opposite(props.direction)};
```

Comment:
Do we really need the `direction` here? Can we maybe use flex to reverse the order?

Reply:
No, as far as i know, this specific functionality only works with the `direction` property (so that the scrollStart is at the opposite direction)